Wrappers/Python/cil/optimisation/algorithms/Algorithm.py

```diff
@@ -134,26 +125,54 @@ def _update_previous_solution(self):
         pass
 
     def get_output(self):
-        " Returns the current solution. "
+        r""" Returns the current solution. 
+        
+        Returns
+        -------
+        DataContainer
+            The current solution 
+             
+        """
         return self.x
 
     def _provable_convergence_condition(self):
+        r""" Checks if the algorithm meets a mathematical convergence criterion.
```

Comment:
Is it the algorithm itself or the current iterative that is checked? If the latter update docstring.

Reply:
I have tried to make this clearer by specifying algorithm set-up